cli.py

```diff
@@ -49,7 +49,7 @@
 
     # In the future, write markdown report or directly push to discussion.
     if args.output_format == "markdown":
-        rendered_report = report.to_markdown(template="github")
+        rendered_report = report.to_markdown(template="huggingface")
```

Comment:
We will need next release after merging https://github.com/Giskard-AI/giskard/pull/1656